Sources/SwiftDocC/Model/Rendering/Diffing/RenderNode+Diffable.swift

```diff
@@ -0,0 +1,55 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021-2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+extension RenderNode: RenderJSONDiffable {
+    
+    /// Returns the differences between this RenderNode and the given one.
+    public func difference(from other: RenderNode) -> JSONPatchDifferences {
```

Comment:
Since we're still viewing diffing as an experimental, in-progress feature, I think it would be good to initially land this as an underscored API. That indicates to the client that while this available to use – it might be removed or changed in the future.

Then, when we feel ready for this feature to graduate to a fully supported one, we can introduce a new non-underscored version of the API and deprecate the underscored version.

What do you think?

```suggestion
    public func _difference(from other: RenderNode) -> JSONPatchDifferences {
```

Reply:
Sounds like a good idea to me, as long as that's the convention followed for experimental features. I will make that change.